app/controllers/site_controller.rb

```diff
@@ -1,4 +1,5 @@
 class SiteController < ApplicationController
   def index
+  	@cities = City.all
```

Comment:
You don't iterate through these cities in `views/site/index.html`, but you should.